library/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.google.maps.android">
```

Comment:
Same here, curious why the package attribute should be dropped.

Reply:
See comment [above](https://github.com/googlemaps/android-maps-utils/pull/1124#discussion_r1041903215).